### PR TITLE
Add detailed guide about how to integrate vscode with docker

### PR DESCRIPTION
**Pull request by xiaofan-luan**
Add some guides about how to development with host + docker environment
Signed-off-by: xiaofan-luan <xiaofan.luan@zilliz.com>

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -157,6 +157,59 @@ The scripts directly under [`build/`](.) are used to build and test. They will e
 
 The `builder.sh` is execute by first creating a “docker volume“ directory in `.docker/`. The `.docker/` directory is used to cache the third-party package and compiler cache data. It speeds up recompilation by caching previous compilations and detecting when the same compilation is being done again.
 
-## CROSS DEBUGGING
-Need detailed documentation about how to cross debugging between host machine IDE and Dev containers, TODO.
+## Debug on Host Machine
+
+### integrate vscode with docker
```
Comment (scsven) [inline build/README.md:162]: The title should start with a capital letter

File: build/README.md [hunk 2]
```diff
@@ -157,6 +157,59 @@ The scripts directly under [`build/`](.) are used to build and test. They will e
...
+
+* Install [Visual Studio Code](https://code.visualstudio.com/)
+
+* Install [Remote Development extension pack]( Remote Development extension pack)
```
Comment (scsven) [inline build/README.md:166]: Is it an incomplete link?

File: build/README.md [hunk 3]
```diff
@@ -157,6 +157,59 @@ The scripts directly under [`build/`](.) are used to build and test. They will e
...
+
+The working principle is as follows: mount the local file system to the workspace inside the container, or copy it to the container. The extension of vs code is installed inside the container and runs in it, so that the vs Code of the host can fully access the tools, platforms and file systems inside the container. This means that you just need to connect to different containers to switch the entire development environment seamlessly.
+
+![image](../docs/imgs/vscode.png)
+
+Taking the Milvus project as an example, there is a file named **.devcontainer.json** in the root directory of the project. This file describes how vs code accesses (or creates) a development container environment, and defines the container environment, working directory, extension tool set, etc.
+
+* The steps to configure the development environment are as follows:
+
+Start VS Code，in the command panel ( F1 ) input **“Remote-Containers: Open Folder in Container”** , then select the project folder which contains devcontainer.json file.
```
Comment (scsven) [inline build/README.md:178]: Change to a halfwidth comma.

File: build/README.md [hunk 4]
```diff
@@ -157,6 +157,59 @@ The scripts directly under [`build/`](.) are used to build and test. They will e
...
+
+or click right-bottom corner button > <  , choose **“Remote-Containers: Open Folder in Container”**，then select the project folder which contains devcontainer.json file.
```
Comment (scsven) [inline build/README.md:180]: Change to a halfwidth comma.

File: build/README.md [hunk 5]
```diff
@@ -157,6 +157,59 @@ The scripts directly under [`build/`](.) are used to build and test. They will e
...
+
+
+![image](../docs/imgs/remote.png)
+
+
+VS Code begin load and construct Devcontainer，the progress bar display the construction state.
```
Comment (scsven) [inline build/README.md:186]: Change to a halfwidth comma.

**Review by wxyucs — CHANGES_REQUESTED** [review]

**Pull request merged**